apps/docs/content/getting-started/react.mdx

```diff
@@ -41,6 +41,7 @@ root.render(
 You will also need to import the Hopper styles in your project. You can do this by importing the following CSS files:
 
 ```css
+@import "@hopper-ui/styled-system/index.css";
```

Comment:
This is something we could discuss eventually. Asking to import that much css seems fishy